src/styles/navbar.css

```diff
@@ -0,0 +1,47 @@
+@import url('https://fonts.googleapis.com/css2?family=Doppio+One&display=swap');
+
+.navbar{
+    height: 10vh;
+    width: 100%;
+    display: flex;
+    align-items: flex-start;
+    justify-content: space-around;
+    font-family: 'Doppio One', 'sans-serif';
+    border-bottom: 1px solid orange;
+}
+
+.logo{
+    height: 80%;
+    width: 10%;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    font-size: 30px;
+    color: orange;
+}
+
+.nav-items{
+    height: 80%;
+    width: 60%;
+    display: flex;
+    align-items: center;
+    justify-content: space-around;
+}
+
+.nav-item{
+    height: 70%;
+    width: 10%;
+    text-decoration: none;
+    color: black;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
+.nav-item:hover{
+    border-bottom: 3px solid orange;
+}
+
+.nav-item-selected{
+    border-bottom: 3px solid orange;
+}
```

Comment:
Please run prettier to fix linting issues

Reply:
FIxed the linting errors